src/Core/Settings.h

```diff
@@ -1013,6 +1013,7 @@ class IColumn;
     M(Bool, input_format_parquet_case_insensitive_column_matching, false, "Ignore case when matching Parquet columns with CH columns.", 0) \
     M(Bool, input_format_parquet_preserve_order, false, "Avoid reordering rows when reading from Parquet files. Usually makes it much slower.", 0) \
     M(Bool, input_format_parquet_filter_push_down, true, "When reading Parquet files, skip whole row groups based on the WHERE/PREWHERE expressions and min/max statistics in the Parquet metadata.", 0) \
```

Comment:
Should `input_format_parquet_filter_push_down` control both min/max filtering and bloom filters or we should keep it separate? @filimonov @al13n321

Reply:
cc @Selfeer